changelog.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [Unreleased]
+
+### Added
+
+- Add `getModels` function in `TerraformParser` to get the list of folder paths that represent a model.
```

Comment:
come from plugin-core 0.16.0, remove this line

Reply:
Done